test/quicapitest.c

```diff
@@ -1230,6 +1233,12 @@ static int test_client_auth(int idx)
                                              &clientquic, NULL, NULL)))
         goto err;
 
+    if (idx > 1) {
+        if (!TEST_true(ssl_ctx_add_large_cert_chain(libctx, cctx, ccert))
+            || !TEST_true(ssl_ctx_add_large_cert_chain(libctx, sctx, cert)))
```

Comment:
Probably this is a comment for @t8m's PR - but I'd really like to know whether this passes with the noisy flag set.